### PR TITLE
Migrate Jenkins Wiki Content to GitHub README.md

### PR DESCRIPTION
**Pull request by eshack94**
This PR is to migrate the plugin documentation from the Jenkins Wiki to the GitHub repo's `README.md`.

For reasoning and context, please see:
- Tweet: https://twitter.com/agentdero/status/1184626102943141888?s=21
- Google Groups discussion: https://groups.google.com/forum/#!msg/jenkinsci-dev/lNmas8aBRrI/eL3u7A6qBwAJ

Commits in this PR:
* Renamed `README` to `README.md`.
* Converted source documentation from https://wiki.jenkins.io/display/JENKINS/Timestamper into Markdown format in `README.md`.
* Updated `pom.xml` to reflect the new documentation location, in accordance with https://wiki.jenkins.io/display/JENKINS/Hosting+Plugins#HostingPlugins-Referencingthedocumentationpagefromtheprojectfile

Please let me know if there are any issues.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+
+### Release History
+
+New releases may take a few hours to appear in the update center.
+
+#### 1.10 and newer
+
+See the [GitHub release page](https://github.com/jenkinsci/timestamper-plugin/releases).
+
+#### 1.9 (2019-02-07)
```
Comment (zbynek) [inline README.md]: Releases prior to 1.10 could go to CHANGELOG.md and only be linked from the readme.

**Comment by eshack94** [comment]
> Great! I left some comments with ideas for further improvements, feel free to accept/reject.

Thanks for the feedback! Making changes and I'll update the PR!

File: README.md (hunk 1 above)
Comment (eshack94) [reply README.md]: @zbynek Added CHANGELOG.md, and migrated release notes to there. Also reference releases page for new release notes going forward.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,346 @@
+# Jenkins Timestamper Plugin
+
+The Timestamper Plugin adds timestamps to the Console Output of Jenkins jobs.
+
+View Timestamper [on the plugin site](https://plugins.jenkins.io/timestamper) for more information.
```
[the remark this answers is not in the document]
Comment (eshack94) [reply README.md]: This change has been made. Thanks for the feedback.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
+# Jenkins Timestamper Plugin
+
+[![Jenkins Plugins](https://camo.githubusercontent.com/c40175c3a041f51ef54a8720643e53abf04a3d60/68747470733a2f2f696d672e736869656c64732e696f2f6a656e6b696e732f706c7567696e2f762f74696d657374616d706572)](https://plugins.jenkins.io/timestamper)
```
Comment (zbynek) [inline README.md]:
```suggestion
[![Jenkins Plugins](https://img.shields.io/jenkins/plugin/v/timestamper)](https://plugins.jenkins.io/timestamper)
```
The `camo.githubusercontent.com` URL is just a cheched version by GitHub, for the badge to show the right version it should use shields.io.

Comment (eshack94) [reply README.md]: My mistake. Changed.

**Comment by zbynek** [comment]
LGTM. @jglick @basil could you please check/merge this?

**Comment by basil** [comment]
> LGTM. @jglick @basil could you please check/merge this?

Hey Zbynek, thanks for reviewing this change. I'd be happy to check and merge this, but I don't (yet!) have permissions to this repository, despite requesting them in jenkins-infra/repository-permissions-updater#1308 (which Jesse has already approved). I'm really not sure what the holdup is. Anyway, once that is resolved I'd be happy to take a look at this.

**Pull request merged**